tests/generators/transition/main.py

```diff
@@ -0,0 +1,41 @@
+from importlib import reload
+from typing import Iterable
+
+from eth2spec.test.helpers.constants import PHASE0, ALTAIR, MINIMAL, MAINNET
+from eth2spec.config import config_util
+from eth2spec.test.altair.transition import test_transition as test_altair_transition
+from eth2spec.phase0 import spec as spec_phase0
+from eth2spec.altair import spec as spec_altair
+
+from eth2spec.gen_helpers.gen_base import gen_runner, gen_typing
+from eth2spec.gen_helpers.gen_from_tests.gen import generate_from_tests
+
+
+def create_provider(tests_src, config_name: str, fork_name: str) -> gen_typing.TestProvider:
+
+    def prepare_fn(configs_path: str) -> str:
+        config_util.prepare_config(configs_path, config_name)
+        reload(spec_phase0)
+        reload(spec_altair)
+        return config_name
+
+    def cases_fn() -> Iterable[gen_typing.TestCase]:
+        return generate_from_tests(
+            runner_name='transition',
+            handler_name='transition',
```

Comment:
can this just be blank `''` to skip the extra level in the file tree? not sure it makes sense to have multiple `handlers` right now